### ISSUE TITLE
[A11y] 4.1.2 Name, Role, Value – Cannot Access All Content With Dragon Naturally Speaking

### ISSUE THREAD

**Issue by dalonsoa** [research-software-directory/RSD-as-a-service#1758]
See https://github.com/ImperialCollegeLondon/RSD-as-a-service/issues/51 for details
